mail-integration-api/src/main/java/org/exoplatform/mailintegration/service/MailIntegrationService.java

```diff
@@ -84,9 +84,10 @@ MessageRestEntity getMessageById(long mailIntegrationSettingId,
                                    long userIdentityId) throws IllegalAccessException;
 
   /**
+   * Update mail integration setting
    *
    * @param mailIntegrationSetting {@link MailIntegrationSetting} object to update
-   * @return updated an {@link MailIntegrationSetting} object
+   * @return updated {@link MailIntegrationSetting} object
    */
-  MailIntegrationSetting updateMailIntegrationSetting(MailIntegrationSetting mailIntegrationSetting) throws IllegalAccessException;
+  MailIntegrationSetting updateMailIntegrationSetting(MailIntegrationSetting mailIntegrationSetting);
```

Comment:
keep the IllegalAccessException to be catched when the user is not authorized to update